src/views/Products/MaterialInformationTouchableBlock.js

```diff
@@ -34,27 +34,30 @@ export default class MaterialInformationTouchableBlock extends Component {
     */
     getArrowIcon(){
         const arrowIcon = this.state.visible
-            ? <Image style={[navStyles.icon, styles.icon]}
+            ? <Image style={navStyles.icon}
                      source={require('./img/arrow_opened.png')}
               />
-            : <Image style={[navStyles.icon, styles.icon]}
+            : <Image style={navStyles.icon}
                      source={require('./img/arrow.png')}
               />;
         return arrowIcon;
     }
     /**
     * @return listMaterial[...]
     */
-    listItem() {
-        const listItem = <View>
-                            <Text style={globalStyles.h4}>
-                              Bruksområde
-                            </Text>
-                            <Text style={globalStyles.h4}>
-                              Bruksområde
-                            </Text>
-                        </View>;
-        return listItem;
+    listInfoDescr() {
+        const listInfoDescr =
+          <View style={styles.infoDescContainer}>
```

Comment:
Remove `const listInfoDescr` 
``` 
return(
<View style={styles.infoDescContainer}>
 </View>
);
```
You don't need to save the JSX in a variable. just return the JSX code in the method